gemma-web/src/main/java/ubic/gemma/web/controller/search/GeneralSearchController.java

```diff
@@ -208,6 +213,15 @@ public ModelAndView search(
                     }
                 }
             }
+
+            if (!noRedirect) {
```

Comment:
This restriction should be restricted to search that are exclusive to EEs. I have some code in the "quick redirect" branch to parse the scope in a nicer way.